packages/esm-patient-chart-app/src/index.ts

```diff
@@ -32,12 +32,17 @@ function setupOpenMRS() {
   registerBreadcrumbs([
     {
       path: spaBasePath,
-      title: 'Patient',
+      // @ts-ignore
+      title: () => window.i18next.getFixedT(null, moduleName)('patientBreadcrumb', 'Patient'),
       parent: `${window.spaBase}/home`,
     },
     {
       path: `${spaBasePath}/:view`,
-      title: ([_, key]) => `${decodeURIComponent(key)} dashboard`,
+      title: ([_, key]) =>
+        `${window.i18next.getFixedT(null, moduleName)(decodeURIComponent(key))} ${window.i18next.getFixedT(
```

Comment:
I don't think this is a correct translation of what was happening before... PS, there's no value in using `getFixedT()` over `t` in this case...

Reply:
I agree, thanks for the review!